components/Login/Login.js

```diff
@@ -1,69 +1,28 @@
-import { useSession, signIn, signOut } from "next-auth/react";
+import { useSession, signIn } from "next-auth/react";
 import LoginIcon from "@/public/icons/LoginIcon.svg";
 import LogoutIcon from "@/public/icons/LogoutIcon.svg";
 import styled from "styled-components";
-import { useEffect, useState } from "react";
-import RegularButton from "../Buttons/RegularButton";
-import ButtonWrapper from "../Buttons/ButtonWrapper";
 
 export default function Login({
   position,
```

Comment:
since your Login element is not bound to be in either two paces anymore, you could rename this prop to "variant" and use "icon" or "expanded" as the two ways to display this component.